keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -288,6 +288,8 @@ class SqlDeliveryConfigRepository(
             .onDuplicateKeyUpdate()
             .set(CURRENT_CONSTRAINT.CONSTRAINT_UID, MySQLDSL.values(CURRENT_CONSTRAINT.CONSTRAINT_UID))
             .execute()
+
+          state.uid = uid
```

Comment:
does this do anything? I don't see state being used again in this method

Reply:
Yes, it updates the `uid` in the original object passed in, so it can be used by the caller. I use that to generate a unique callback ID to send to echo, which later gets sent back to keel and allows us to retrieve the corresponding `ConstraintState` to update with the user's approval/rejection.